common/pom.xml

```diff
@@ -62,6 +62,10 @@
             <groupId>jakarta.activation</groupId>
             <artifactId>jakarta.activation-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>jakarta.ws.rs</groupId>
+            <artifactId>jakarta.ws.rs-api</artifactId>
```

Comment:
Minor: I think this change means that we can probably cleanup this `jakarta.ws.rs-api` dependency from some other files, which depend on `keycloak-common`, which is pretty much everything. For example `services/pom.xml` .

When I currently run:
```
cd services
mvn dependency:tree
```
the output contains:
```
jakarta.ws.rs:jakarta.ws.rs-api:jar:3.1.0:compile
```
I think that with your change to `common/pom.xml` , the dependency can be removed from the `services/pom.xml` and the `mvn dependency:tree` would still show the `jakarta.ws.rs:jakarta.ws.rs-api:jar:3.1.0:compile` (due the services have transitive dependency on `jakarta.ws.rs-api` through `keycloak-common`. Maybe some other places can be cleaned as well in relation to this. WDYT?

Reply:
I've removed `jakarta.ws.rs-api` everywhere it was already a transitive dependency, through `common`, or otherwise.